spec/std/callstack_spec.cr

```diff
@@ -17,9 +17,9 @@ describe "Backtrace" do
     output = `#{tempfile.path}`
 
     # resolved file line:column
-    output.should match(/#{sample} 3:10/) # callee1
-    output.should match(/#{sample} 15:3/) # callee3
-    output.should match(/#{sample} 17:1/) # ???
+    output.should match(/callee1 at #{sample} 3:10/)
+    output.should match(/callee3 at #{sample} 15:3/)
```

Comment:
Oh, I see. 

Since in OSX the output of `backtrace_sample` changes before and after the use of `dsymutil` I though it was working on OSX.

So, Is this a WIP still? 
Is the osx dwarf parsing possible? 
Is there anything someone could help with to get that part done?

```
$ ./bin/crystal build --debug spec/std/data/backtrace_sample -o ./backtrace_sample

$ ./backtrace_sample 
0x104731675: *CallStack::unwind:Array(Pointer(Void)) at ??
0x104731611: *CallStack#initialize:Array(Pointer(Void)) at ??
0x1047315e8: *CallStack::new:CallStack at ??
0x104730765: *caller:Array(String) at ??
0x1047760e1: *Kls#callee1:Nil at ??
0x104730741: *callee3:Nil at ??
0x1047207d7: __crystal_main at ??
0x104730618: main at ??

$ dsymutil --flat ./backtrace_sample 

$ ./backtrace_sample 
0x109bfe0e1: *Kls#callee1:Nil at /.../src/spec/std/data/backtrace_sample 3:10
0x109bb8741: *callee3:Nil at /.../src/spec/std/data/backtrace_sample 15:3
0x109ba87d7: __crystal_main at /.../src/spec/std/data/backtrace_sample 17:1
0x109bb8618: main at /.../src/main.cr 12:15

```